aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -100,19 +100,19 @@ The tutorial set you've just started covers these features of ASP.NET Web Pages:
 
 This tutorial set and subsequent ones revolve around a website where you can list movies that you like. You'll be able to enter movies, edit them, and list them. Here are a couple of the pages you'll create in this tutorial set. The first one shows the movie listing page that you'll create:
 
-![Finshed Movie app showing a movie listing](getting-started/_static/image1.png)
+![Screenshot of the My Movie Site application showing a movie listing, Search Genre and Search Title buttons, and search fields.](getting-started/_static/image1.png)
 
 And here's the page that lets you add new movie information to your site:
 
-![Finished movie app showing the Add Movie page](getting-started/_static/image2.png)
+![Screenshot of the finished movie app showing the Add Movie page with Title, Genre, and Year text fields.](getting-started/_static/image2.png)
```

Comment:
Don't change valid alt-text

Reply:
Alt text was modified to reflect Microsoft’s accessibility guidelines for alt text per https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main.